server/schedule/region_scatterer_test.go

```diff
@@ -433,16 +433,15 @@ func TestScatterForManyRegion(t *testing.T) {
 
 	scatterer := NewRegionScatterer(ctx, tc, oc)
 	regions := make(map[uint64]*core.RegionInfo)
-	for i := 1; i <= 15000; i++ {
-		regions[uint64(i)] = tc.AddLightLeaderRegion(uint64(i), 1, 2, 3)
+	for i := 1; i <= 150; i++ {
```

Comment:
Why change it?

Reply:
for CI test, had change it back to 15000